precode.sh

```diff
@@ -6,38 +6,43 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p /tmp/task/dir3/dir4
```

Comment:
С помощью флага `-p` можно сделать все в одну команду
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```

Reply:
Так же чтобы не добавлять везде `/tmp`, то можно в начале просто перейти. Если потом нужно будет поменять папку, то просто ее поменять при переходе, а не в 4 командах